community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -318,10 +316,31 @@ def ignore_err(e) -> bool:
         if failures:
             log.error(f"some placement groups failed to delete: {failures}")
     log.info(
-        f"deleted {len(done)} of {len(placement_groups)} placement groups ({to_hostlist(done.keys())})"
+        f"deleted {len(done)} of {len(links)} placement groups ({to_hostlist(done.keys())})"
     )
 
 
+
+@lru_cache
+def _get_resource_policies_in_region(lkp: util.Lookup, region: str) -> list[Any]:
+    res = []
+    act = lkp.compute.resourcePolicies()
+    op = act.list(project=lkp.project, region=region)
+    prefix = f"{lkp.cfg.slurm_cluster_name}-slurmgcp-managed-"
+    while op is not None:
+        result = ensure_execute(op)
+        res.extend([p for p in result.get("items", []) if p.get("name", "").startswith(prefix)])
+        op = act.list_next(op, result)
+    return res
+
+
+@lru_cache
+def _get_resource_policies(lkp: util.Lookup) -> list[Any]:
```

Comment:
Should be uppercase List?

Reply:
nope, since python X one can use builtin keywords instead of `typing.List|Set|Dict|...`